app/views/idv/shared/_document_capture.html.erb

```diff
@@ -142,7 +142,17 @@
 
   <p class='margin-top-4 margin-bottom-0'><%= t('doc_auth.info.document_capture_upload_image') %></p>
 </div>
-<%= render 'idv/doc_auth/start_over_or_cancel' %>
+
+<%= render 'idv/doc_auth/start_over' %>
```

Comment:
Sorta out-of-scope, but it occurs to me: What purpose is "Start Over" meant to have in the hybrid session? Right now it takes them back to the login screen, which is more-or-less the same as if they'd click "Cancel", except we don't actually flag it as a cancellation. 

Maybe we should only show the "Cancel" link in hybrid sessions? I imagine that might avoid the need for the refactoring of `_start_over_or_cancel.html.erb` to split the two links.

Otherwise, we could create a ticket to figure out what to do with the "Start Over" link.